src/discussion-tool-selector/DiscussionToolSelector.scss

```diff
@@ -0,0 +1,16 @@
+.discussion-tool {
+    &:hover,
+    &:focus {
+        border-color: theme-color("primary", "hover") !important;
+    }
+}
+
+.features-table {
+    scroll-snap-type: x mandatory;
+
+    th {
+        min-width: 7.5rem;
+        white-space: nowrap;
```

Comment:
This makes the column stretch too long, remove this so that you can have word breaks. 

```suggestion
```